### DISCUSSION TITLE
multiple ASN "groups"

### DISCUSSION THREAD

**Discussion by benjaminfrank** [jonaswinkler/paperless-ng#802] [Feature Requests]
Hi, I recently started to use paperless-ng. Let me first say a big thank you for the software!

I am currently looking into the ASN part of paperless-ng. I like the whole idea behind ASN but can't yet do to the required mental adjustment for parts of my paperwork. I feel a bit uneasy if I cannot quickly locate them or maybe more important cannot tell someone else how to locate them without paperless-ng  , e.g., certain insurances.

As possible solution would be two or more separate ASNs so that I could separate those papers into a dedicated filer.
I though about an (optional) prefix for ASNs. This way the standard use-case is still work exactly like it does now but for special use cases one could use a dedicated ASN by prefixing it.  I was thinking of a single character to have it visually set aside from the ASN part. With this locating the document becomes much easier in case you have a lot of paperwork.

Would this be something that could be considered useful for others as well?

**Comment by Boldie** [comment]
I already stumpled around your concern too. What shall I do if there is nothing online and I have to look around in a lot of physical folders. I am thinking of  a printable index which is like printing Tag x sorted by date and then add these tables from time to time to my folders. I bought a paginator numbering stamp for the numbers, so I am unable to make multiple numbers and I also do not want to track the groups around, which implies new errors. I think a printable index will solve my concerns. Maybe this is also a solution for you. Unfortunately: I am starting with paperless and have not finished this index printing now :)

**Reply by Boldie** [reply]
Ahh scrolling down the list I found #126 which is quite what I have described for now.

**Comment by gahsul** [comment]
I would also need a second ASN. I would like not to mix my physical documents with those of my girlfriend. I prefer to have the physical documents separated in two Binders.

As a workaround I use negative ASNs for my girlfriend, but I don't know if that could somehow confuse Papperless in the future.

**Comment by Ycto** [comment]
Personally, I also prefer to physically separate documents by (broad) category. Currently, I'm "abusing" the document type feature for this: I define each physical folder/binder as a document type and order the documents by ASN within those binders. This works pretty well, it's just a bit "ugly" as "folder" would be a more correct term. 

Maybe an extra "folder" metadata field is a solution here? An added advantage is that existing archives can either remain unchanged or are very easy to adapt. The only obvious annoyance is that within binders, the document ASN will have "gaps" in their sequence (but they can still be ordered).

**Comment by philpagel** [comment]
In my situation, I have two fundamentally different classes of documents (private vs. company) which I want to keep (physically) separate. I have solved this by assigning two separate ASN numeric ranges: 0-99999 for _private_ and 100000–199999 for _company_. I am using two separate automatic number stamps and separate binders.

**Comment by krehl** [comment]
I have a similar use case as I also want to have unique numbers for purely digital documents like invoices submitted as PDFs.
